### ISSUE TITLE
List projects using fusejs in README.

### ISSUE THREAD

**Issue by LopezCatherine** [c4milo/fusejs#18]
This would help newcomers to get an idea of where it is being used any how it is used.

**Comment by thejinx0r** [comment]
I agree, but I think it would help if there were examples ( #1 and #2 ).
I think a full blown project would be overwhelming to a new user.

Since nodejs/iojs is path based and the low level api is inode based, it's a bit more complicated to write a simple loopback filesystem, but I will work on it.

Example filesystems will be coming. The only one I know for sure that's using it is 
https://github.com/thejinx0r/node-gdrive-fuse which is the project that is making use of it.
